src/pilot/assign_role.py

```diff
@@ -308,10 +308,10 @@ def define_controller_logical_disks(drac_client, raid_controller_name):
     raid_10_logical_disk = define_single_raid_10_logical_disk(
         drac_client, raid_controller_name)
 
-    if raid_10_logical_disk is None:
-        return None
-
     logical_disks = list()
+    if not raid_10_logical_disk:
```

Comment:
See my comment below in define_compute_logical_disks.  The same holds true here.

Reply:
Done the changes. Please review it.